pkg/cmd/server/admission/init.go

```diff
@@ -38,3 +40,19 @@ func Validate(plugins []admission.Interface) error {
 	}
 	return nil
 }
+
+// GetPluginConfigFile translates from the master plugin config to a file name containing
+// a particular plugin's config (the file may be a temp file if config is embedded)
+func GetPluginConfigFile(pluginConfig map[string]api.AdmissionPluginConfig,
+	pluginName string, defConfigFile string) (string, error) {
```

Comment:
Is `defConfigFile` the content of the file or its location?  (adjust the name to indicate what it is).

Also, that seems a little early for a line break, 80 characters is a relic.